content/python/concepts/math-module/terms/math-comb/math-comb.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'math.comb()'
+Description: 'Finds out how many unordered choices can be made in order to choose "k" amount of items from "n" number of items.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Math'
+  - 'Combinations'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+
+**Math.comb()** function allows you to find out how many unordered choices can be possibly made in order to choose 'k' amount of items from 'n' number of items.
+
+## Syntax
+
+```pseudo
+math.comb(n, k)
+```
+where: 
+
+    n, k - non-negative integers
```

Comment:
```suggestion
- `n`: A non-negative integer that represents the total number of items.
- `k`: A non-negative integer that represents the number of items to choose.
```

Reply:
Elaborated both the parameters in the required format:
- `parameter1`: its description
- `parameter2`: its description